logformat.md

```diff
@@ -0,0 +1,51 @@
+# Gazebo Log Format Update
+
+## Overview
+
+The Gazebo log format is an SQLite3 database that stores serialized protobuf
+messages along with topic names in a time sequence. Importantly, simulation
+state information is recorded as serialized Components. This relies on
+a unique type id to associate the data to a component type. This type
+information is available via c++ typeid, which is not readily
+cross-language. For example, a javascript logfile reader would not know how
+to associate a typeid with the correct component.
+
+
+State information encoded as components requires a log file reader to
+implement or use Gazebo’s ECS. This represents a large barrier to entry.
+Ideally, a user could query data from a log file without implementing
+a complete ECS. 
+
+## Requirements
+
+1. Cross-platform and cross-language
```

Comment:
I think we should note a minimum set of languages to help bound this a little bit.

From what we use in Gazebo, (C++, Ruby, Python, Typescript/Javascript, Go) would cover 99% of our use cases correct?